hack/jenkins/e2e.sh

```diff
@@ -472,6 +472,21 @@ case ${JOB_NAME} in
     : ${ENABLE_DEPLOYMENTS:=true}
     ;;
 
+  kubernetes-e2e-gke-subnet)
+    : ${DOGFOOD_GCLOUD:="true"}
+    : ${E2E_CLUSTER_NAME:="jkns-gke-subnet"}
+    : ${E2E_NETWORK:="auto-subnet"}
```

Comment:
Can you put the gcloud command flags or mention what checkbox(s) to tick on ui? If a month from now it gets deleted, good chance the person fixing the test won't know what options to recreate it with.